src/test/java/com/google/cloud/spanner/pgadapter/ITParameterizedQueryTest.java

```diff
@@ -66,20 +67,24 @@ public void setUp() throws Exception {
     Database db = testEnv.createDatabase();
     testEnv.updateDdl(db.getId().getDatabase(), Arrays.asList(ddl));
     testEnv.updateTables(db.getId().getDatabase(), Arrays.asList(dml));
-    String[] args = {
-      "-p",
-      testEnv.getProjectId(),
-      "-i",
-      testEnv.getInstanceId(),
-      "-d",
-      db.getId().getDatabase(),
-      "-c",
-      testEnv.getCredentials(),
-      "-s",
-      String.valueOf(testEnv.getPort()),
-      "-e",
-      "staging-wrenchworks.sandbox.googleapis.com"
-    };
+    String credentials = testEnv.getCredentials();
+    ImmutableList.Builder<String> argsListBuilder =
+        ImmutableList.<String>builder()
+            .add(
+                "-p",
+                testEnv.getProjectId(),
+                "-i",
+                testEnv.getInstanceId(),
+                "-d",
+                db.getId().getDatabase(),
+                "-s",
+                String.valueOf(testEnv.getPort()),
+                "-e",
+                testEnv.getUrl().getHost());
+    if (credentials != null) {
```

Comment:
Do not require credentials to be specified in the arguments. This enables the usage of the default credentials of the environment.